<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UartConsoleWidget</class>
 <widget class="QFrame" name="UartConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uart Console</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QTextBrowser" name="textActivity">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rbtnRxEditAscii">
     <property name="text">
      <string>ascii</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">groupRxEditFormat</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="rbtnRxEditHex">
     <property name="text">
      <string>hex</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">groupRxEditFormat</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>send on enter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cboxSendOnEnter">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CR-LF</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="btnRxError">
     <property name="text">
      <string>error</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="editRx"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="groupRxEditFormat"/>
 </buttongroups>
</ui>
